docs/cloud-native-security/d4c-policy-guide.asciidoc

```diff
@@ -0,0 +1,94 @@
+
+[[d4c-default-policies]]
+[discrete]
+== Default policies:
+The default D4C integration policy includes two selector-response pairs. It is designed to implement core container workload protection capabilities:
+
+- *Threat Detection:* The first selector-response pair is designed to stream process telemetry events to your {es} cluster, so that {elastic-sec} can evaluate it to detect threats. Both the selector and response are named `allProcesses`. The selector selects all fork and exec events. The associated response specifies that the selected events should be logged.
+- *Drift Detection & Prevention:* The second selector-response pair is designed to create alerts when container drift is detected. Both the selector and response are named `executableChanges`. The selector selects all `createExecutable` and `modifyExecutable` events. The associated response specifies that the selected events should create alerts, which will be sent to your {es} cluster. You can modify the response to block drift operations by setting it to block.
+
+image::images/d4c-policy-editor.png[The defend for containers policy editor with the default policies]
+
+
+[[d4c-selectors-glossary]]
+[discrete]
+== Selectors
+A selector requires a name and at least one operation. It will select all events of the specified operation types, unless you also include _conditions_ to narrow down the selection. Some conditions are available for both `file` and `process` selectors, while some are only available for one type of selector.
+
+[discrete]
+=== Common conditions
+These conditions are available for both `file` and `process` selectors.
+
+[cols="1,1", options="header"]
+|===
+| Name | Description
+| containerImageFullName | A list of container full image names to match on, for example: `docker.io/nginx`.
+| containerImageName | A list of container image names to match on. For example: `nginx`.
+| containerImageTag | A list of container image tags to match on. For example: `latest`.
+| kubernetesClusterId | A list of Kubernetes cluster IDs to match on. For consistency with KSPM, the `kube-system` namespace's UID is used as a cluster ID.
+| kubernetesClusterName | A list of Kubernetes cluster names to match on.
+| kubernetesNamespace | A list of Kubernetes namespaces to match on.
+| kubernetesPodName | A list of Kubernetes pod names to match on. Trailing wildcards supported.
+| kubernetesPodLabel | A list of resource labels. Trailing wildcards supported (value only), for example: `key1:val*`.
+|===
+
+[discrete]
+=== File-selector conditions
+These conditions are available only for `file` selectors.
+
+[cols="1,1", options="header"]
+|===
+| Name | Description
+| operation | The list of system operations to match on. Options include `createExecutable`, `modifyExecutable`, `createFile`, `modifyFile`, `deleteFile`.
+| ignoreVolumeMounts | If set, ignores file operations on ALL volume mounts.
```

Comment:
Made a slight style edit in case you typed "all" in caps to emphasize it:

```suggestion
| ignoreVolumeMounts | If set, ignores file operations on _all_ volume mounts.
```